### PR TITLE
./scripts/run: provision tmp/cloud-config.yml as cloud-config

### PR DESCRIPTION
**Pull request by imikushin**
./scripts/run: provision tmp/cloud-config.yml as cloud-config

### CODE REVIEW

File: scripts/run [hunk 1]
```diff
@@ -56,13 +56,17 @@ fi
...
 mkdir -p $(dirname $USER_DATA)
 
-echo "#cloud-config" > ${USER_DATA}
-echo "ssh_authorized_keys:" >> ${USER_DATA}
-for i in ${HOME}/.ssh/*.pub; do
-    if [ -e $i ]; then
-        echo "  - $(<$i)" >> ${USER_DATA}
-    fi
-done
+if [ -e ../../tmp/cloud-config.yml ]; then
```
Comment (ibuildthecloud) [inline scripts/run]: You want to just take this as an arg to `/scripts/run` instead of a magic path?

Comment (imikushin) [inline scripts/run]: Agreed

Comment (imikushin) [inline scripts/run]: Done

File: scripts/run [hunk 2]
```diff
@@ -17,6 +17,7 @@ INITRD_TMP=${BUILD}/$(sha1sum ${INITRD} | awk '{print $1}')
 INITRD_CURRENT=${BUILD}/initrd-current
 INITRD_TEST=${BUILD}/initrd.test
 USER_DATA=cloud-init/openstack/latest/user_data
+CLOUD_CONFIG=$(readlink -f "$1") || :
```
Comment (ibuildthecloud) [inline scripts/run]: There's already an arg parsing while loop lower in the script.  This will conflict with that.

**Comment by imikushin** [comment]
@ibuildthecloud it's working for me now

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/tmp
```
Comment (ibuildthecloud) [inline .gitignore]: We can get rid of this now

Comment (imikushin) [inline .gitignore]: NP

**Comment by imikushin** [comment]
@ibuildthecloud I hope it's ready to merge.

**Pull request merged**